pilot-app/src/components/form/fields/wimChoiceCheckbox.vue

```diff
@@ -0,0 +1,36 @@
+﻿<template>
+    <div :class="classname">
+        <label v-if="prefix(field)" v-html="prefix(field)"></label>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
        <input type="checkbox"
```